Source/WebInspectorUI/UserInterface/Views/ApplicationCacheDetailsSidebarPanel.js

```diff
@@ -1,165 +0,0 @@
- *    notice, this list of conditions and the following disclaimer in the
- *    documentation and/or other materials provided with the distribution.
- *
- * THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS ``AS IS''
- * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
- * THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
- * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS
- * BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
- * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
- * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
- * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
- * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
- * THE POSSIBILITY OF SUCH DAMAGE.
- */
-
-WI.ApplicationCacheDetailsSidebarPanel = class ApplicationCacheDetailsSidebarPanel extends WI.DetailsSidebarPanel
-{
-    constructor()
-    {
-        super("application-cache-details", WI.UIString("Storage"));
-
-        this.element.classList.add("application-cache");
-
-        this._applicationCacheFrame = null;
-    }
-
-    // Public
-
-    inspect(objects)
-    {
-        // Convert to a single item array if needed.
-        if (!(objects instanceof Array))
-            objects = [objects];
-
-        var applicationCacheFrameToInspect = null;
-
-        // Iterate over the objects to find a WI.ApplicationCacheFrame to inspect.
-        for (var i = 0; i < objects.length; ++i) {
-            if (objects[i] instanceof WI.ApplicationCacheFrame) {
-                applicationCacheFrameToInspect = objects[i];
-                break;
-            }
-        }
-
-        this.applicationCacheFrame = applicationCacheFrameToInspect;
-
-        return !!this.applicationCacheFrame;
-    }
-
-    get applicationCacheFrame()
-    {
-        return this._applicationCacheFrame;
-    }
-
-    set applicationCacheFrame(applicationCacheFrame)
-    {
-        if (this._applicationCacheFrame === applicationCacheFrame)
-            return;
-
-        this._applicationCacheFrame = applicationCacheFrame;
-
-        this.needsLayout();
-    }
-
-    closed()
-    {
-        if (this.didInitialLayout) {
-            WI.applicationCacheManager.removeEventListener(WI.ApplicationCacheManager.Event.NetworkStateUpdated, this._networkStateUpdated, this);
-            WI.applicationCacheManager.removeEventListener(WI.ApplicationCacheManager.Event.FrameManifestStatusChanged, this._frameManifestStatusChanged, this);
-        }
-
-        super.closed();
-    }
-
-    // Protected
-
-    initialLayout()
-    {
-        super.initialLayout();
-
-        this._locationManifestURLRow = new WI.DetailsSectionSimpleRow(WI.UIString("Manifest URL"));
-        this._locationFrameURLRow = new WI.DetailsSectionSimpleRow(WI.UIString("Frame URL"));
```

Comment:
please dont forget to update `Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js` :)